src/Chainweb/Pact/Utils.hs

```diff
@@ -17,22 +17,29 @@ module Chainweb.Pact.Utils
     -- * time-to-live related items
     , maxTTL
     , timingsCheck
+    , fromPactChainId
     ) where
 
 import Data.Aeson
 
 import Control.Concurrent.MVar
+import Control.Monad.Catch
 
 import Pact.Interpreter as P
 import Pact.Parse
-import Pact.Types.Command
+import qualified Pact.Types.ChainId as P
 import Pact.Types.ChainMeta
+import Pact.Types.Command
 
 import Chainweb.BlockHeader
+import Chainweb.ChainId
 import Chainweb.Pact.Backend.Types
 import Chainweb.Time
 import Chainweb.Transaction
 
+fromPactChainId :: MonadThrow m => P.ChainId -> m ChainId
+fromPactChainId (P.ChainId t) = chainIdFromText t
```

Comment:
@larskuhtz You good with this? iirc you had a version of chainweb chainid to textual pact chain id that was what you wanted in a different PR

Reply:
It's actually an "unsafe" construction of `ChainId` but it's fine for now. 

This function should actually be called `uncheckedChainIdFromText` and `chainIdFromText` should be

```haskell
chainIdFromText :: MonadThrow m => HasChainwebVersion v => v -> T.Text -> m ChainId
```

I have a pending branch that locks down construction of chainIds, that I hope to find some time to finish soon.

Reply:
So 👍 for now, since there is still no safe/checked version of `chainIdFromText`.